src/goose/brokers/redis/batch.clj

```diff
@@ -0,0 +1,27 @@
+(ns goose.brokers.redis.batch
+  (:require
+    [goose.brokers.redis.commands :as redis-cmd]
+    [goose.defaults :as d]
+    [goose.job :as job]
+    [goose.retry :as retry]))
```

Comment:
Can we also keep this as goose.retry (as discussed [here](https://github.com/nilenso/goose/pull/124#discussion_r1257292927))